packages/edit-post/src/components/layout/index.js

```diff
@@ -426,7 +428,7 @@ function Layout( {
 				isDistractionFree: get( 'core', 'distractionFree' ),
 				showMetaBoxes:
 					! DESIGN_POST_TYPES.includes( currentPostType ) &&
-					isRenderingPostOnly,
+					( isRenderingPostOnly || isRenderingTemplateLocked ),
```

Comment:
Can you refresh my memory here? Do we have rendering modes besides `post-only` and `template-locked`?

If not, then I think we can remove rendering mode checks from this condition.

Reply:
I am not a 100% certain but I thought that the "edit template" portion in the post editor technically uses `template-only` as the rendering mode 🤔 

@youknowriad may be able to help clarify this though :)

Reply:
Judging from [the comment of the `setRenderingMode` action](https://github.com/WordPress/gutenberg/blob/7714bce8d9356707a020b5d7df84797832c423f8/packages/editor/src/store/actions.js#L651-L652), there should be no modes other than `post-only` and `template-locked`. The `template-only` mode has been removed in #57700. I've tried searching Gutenberg in general for `getRenderingMode()` or `setRenderingMode(`, but I can't find any other modes.

Therefore, I think this check should be removable, but I will ping @stokesman to double check.

Reply:
Removing it looks safe to me. I quickly checked with "edit template" and its rendering mode is "post-only". The condition that matters in that case is the post type being "wp_template".

Reply:
Thanks all! I just updated the code to remove the check entirely  👍 :)